src/foam/u2/view/EmbeddedTableView.js

```diff
@@ -88,7 +95,7 @@ foam.CLASS({
           class: this.DAOBrowseControllerView,
           data$: this.data$,
           config$: this.config$
-        }, parent: this }));
+        }, parent: this.__subContext__.createSubContext({ controllerMode: 'CREATE' }) }));
```

Comment:
wonder why this is necessary.
Don't see any harm in this, but what reason do we have to set CREATE instead of VIEW?

Reply:
https://github.com/kgrgreer/foam3/blob/2e3e2958fd39667bb4bf8796bbedbaab97392215/src/foam/u2/Element.js#L2850-L2860 Views set their displayMode based on controllerMode. Because we push from a RO view (DAOSummaryView which is the detail view) so when we render the DAOBrowseControllerView it inherits the same context which means the controllerMode and hence a RO display mode which is what was causing the search bar to be RO as well. This just resets it to the default 'CREATE' mode. Not sure if the better approach would be to set it to `undefined` and let the factory handle the assignment but we can ask Kevin once he is back.